Resources/Prototypes/Entities/Objects/Specific/Service/vending_machine_restock.yml

```diff
@@ -46,10 +49,59 @@
         acts: [ "Destruction" ]
 
 - type: entity
-  parent: BaseVendingMachineRestock
+  parent: BaseItem
```

Comment:
reparenting all of the existing ones to ours is fine, and adding staticprices to cut on exploits is as well, but similar to the other yml prototypes. Lets put our custom ones in separate files in _nf so it becomes easier to merge/maintain and find our custom items. The modifications to the existing ones in the file are ok